sdk/py/apepay/manager.py

```diff
@@ -87,123 +91,123 @@ def _parse_validator(self) -> Callable[[_ValidatorItem], Validator]:
         return self.set_validators(
             *(set(self.validators) - set(map(self._parse_validator, old_validators))),
             **txn_kwargs,
         )
 
-    @property
-    def add_token(self) -> ContractTransactionHandler:
-        return self.contract.add_token
+    def add_token(self, token: AddressType, **txn_kwargs) -> ReceiptAPI:
+        return self.contract.set_token_accepted(token, True, **txn_kwargs)
 
-    @property
-    def remove_token(self) -> ContractTransactionHandler:
-        return self.contract.remove_token
+    def remove_token(self, token: AddressType, **txn_kwargs) -> ReceiptAPI:
+        return self.contract.set_token_accepted(token, False, **txn_kwargs)
 
-    @property
-    def is_accepted(self) -> ContractCallHandler:
-        return self.contract.token_is_accepted
+    def is_accepted(self, token: AddressType) -> bool:
+        return self.contract.token_is_accepted(token)
 
     @cached_property
     def MIN_STREAM_LIFE(self) -> timedelta:
         # NOTE: Immutable in contract
         return timedelta(seconds=self.contract.MIN_STREAM_LIFE())
 
+    def compute_funding_rate(
+        self,
+        funder: AddressType,
+        token: Any,
+        amount: int,
+        products: list[HexBytes],
+    ) -> int:
+        total = 0
+
+        for validator in self.validators:
+            try:
+                total += validator(funder, token, amount, products)
+            except ContractLogicError:
+                pass
+
+        return total
+
     def create(
         self,
         token: ContractInstance,
-        amount_per_second: str | int,
-        reason: HexBytes | bytes | str | dict | None = None,
-        start_time: datetime | int | None = None,
+        amount: str | int,
+        products: list[HexBytes],
+        min_stream_life: timedelta | int | None = None,
         **txn_kwargs,
     ) -> "Stream":
-        if not self.is_accepted(token):
+        if not self.is_accepted(token.address):  # for mypy
             raise TokenNotAccepted(str(token))
 
-        if isinstance(amount_per_second, str) and "/" in amount_per_second:
-            value, time = amount_per_second.split("/")
-            amount_per_second = int(
-                self.conversion_manager.convert(value.strip(), int)
-                / time_unit_to_timedelta(time).total_seconds()
-            )
-
-        if amount_per_second == 0:
-            raise ValueError("`amount_per_second` must be greater than 0.")
-
-        args: list[Any] = [token, amount_per_second]
-
-        if reason is not None:
-            if isinstance(reason, dict):
-                reason = json.dumps(reason, separators=(",", ":"))
+        if not isinstance(amount, int):
+            amount = self.conversion_manager.convert(amount, int)
+        assert isinstance(amount, int)  # for mypy
 
-            if isinstance(reason, str):
-                reason = reason.encode("utf-8")
+        args: list[Any] = [token, amount, products]
```

Comment:
Having a local variable named `args` is something I try to avoid because `pdb` also has something called `args` so when debugging using `pdb`, it is harder to inspect the variable

Reply:
b8b07d0